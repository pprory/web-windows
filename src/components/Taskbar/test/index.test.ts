import { DeskTop } from "../../index"
import { shallowMount } from '@vue/test-utils'

test("桌面组件测试", () => {
    const wrapper = shallowMount(DeskTop)
    wrapper.find('button').trigger('click').then(_ => {
        console.log(wrapper.text())
        expect(wrapper.html()).toMatchSnapshot()
    })
})