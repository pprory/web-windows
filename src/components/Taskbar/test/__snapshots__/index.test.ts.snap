// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`点击底部导航条按钮切换昵称测试 1`] = `
"<div title=\\"456\\"> 底部任务栏 <button>点击切换昵称</button>
  <p>PP</p>
</div>"
`;
